backend/src/routes/salesRoute.js

```diff
@@ -12,16 +17,11 @@ const __dirname = path.dirname(__filename);
 
 // middleware
 
-salesRouter.use(
-  express.static(path.join(__dirname, "../../../frontend/node_modules"))
-);
-salesRouter.use(
-  express.static(path.join(__dirname, "../../../frontend/public"))
-);
-
 salesRouter.use(bodyParser.json());
 salesRouter.use(bodyParser.urlencoded({ extended: true }));
 
-salesRouter.get("/sales", getSales);
+salesRouter.get("/", verifyToken, isAdmin, getSales);
+salesRouter.post("/", verifyToken, createSale);
+salesRouter.get("/:saleId", verifyToken, isAdmin, getSaleById);
```

Comment:
This limits the endpoint to only the admin, you need to find a way to allow both admin and the creator of the sale to access it. That seems like a new middleware to me.